### ISSUE TITLE
Support Scale to UPM Custom Parameter when exporting from Glyphs to UFO

### ISSUE THREAD

**Issue by marcologous** [googlefonts/fontmake#676]
Building TTFs from within Glyphsapp respects the Scale to UPM Custom Parameter which performs scaling of all relevant values on the generated TTF Binaries.

Doing the same thing with FontMake has no effect. It would be extremely good to have support for this parameter since manually scaling all glyphs in the source files would take a lot of time and effort.

This problem was observed when preparing for a pull request for the Hanken Grotesk family to be added to the Google Fonts collection.

https://travis-ci.org/github/marcologous/hanken-grotesk/builds/704405010#L1354

**Comment by felipesanches** [comment]
I agree. I would be interested in fixing this bug if no one else's already working on it.
@anthrotype, do you have any suggestion on how should I approach an implementation?
